packages/dnb-eufemia/src/components/list-format/__tests__/ListFormat.test.tsx

```diff
@@ -0,0 +1,654 @@
+
+    rerender(
+      <ListFormat variant="ul" listType="square">
+        <React.Fragment key="123">123</React.Fragment>
+        <React.Fragment key="456">456</React.Fragment>
+        <React.Fragment key="789">789</React.Fragment>
+      </ListFormat>
+    )
+    expect(list('ul')).toHaveAttribute('type', 'square')
+  })
+
+  it('formats value in different locale using value', () => {
+    const { container } = render(
+      <Provider locale="en-GB">
+        <ListFormat value={['Baz', 'Bar', 'Foo']} />
+      </Provider>
+    )
+
+    expect(container).toHaveTextContent('Baz, Bar and Foo')
+  })
+
+  it('formats value in different locale using children', () => {
+    const { container } = render(
+      <Provider locale="en-GB">
+        <ListFormat>{['Baz', 'Bar', 'Foo']}</ListFormat>
+      </Provider>
+    )
+
+    expect(container).toHaveTextContent('Baz, Bar and Foo')
+  })
+
+  it('formats value in different locale using children as JSX', () => {
+    const { container } = render(
+      <Provider locale="en-GB">
+        <ListFormat>
+          <>Baz</>
+          <>Bar</>
+          <>Foo</>
+        </ListFormat>
+      </Provider>
+    )
+
+    expect(container).toHaveTextContent('Baz, Bar and Foo')
+  })
+
+  it('formats value in different locale using children as JSX with keys', () => {
+    const { container } = render(
+      <Provider locale="en-GB">
+        <ListFormat>
+          <React.Fragment key="123">Baz</React.Fragment>
+          <React.Fragment key="456">Bar</React.Fragment>
+          <React.Fragment key="789">Foo</React.Fragment>
+        </ListFormat>
+      </Provider>
+    )
+
+    expect(container).toHaveTextContent('Baz, Bar and Foo')
+  })
+})
+
+describe('ListFormat aria', () => {
+  it('should validate', async () => {
+    const Component = render(<ListFormat value={[1]} />)
+    expect(await axeComponent(Component)).toHaveNoViolations()
+  })
+})
+
+describe('listFormat', () => {
+  it('accepts undefined', () => {
+    expect(listFormat(undefined)).toBe(undefined)
+  })
+
+  it('accepts null', () => {
+    expect(listFormat(null)).toBe(null)
+  })
+
+  it('accepts array with undefined', () => {
+    expect(listFormat([undefined])).toBe('')
+  })
+
+  it('accepts array with null', () => {
+    expect(listFormat([null])).toBe('')
+  })
+
+  it('accepts array with false', () => {
+    expect(listFormat([false])).toBe('')
+  })
+
+  it('accepts array with 0', () => {
+    expect(listFormat([0])).toBe('0')
+  })
+
+  it('accepts empty array', () => {
+    expect(listFormat([])).toBe('')
+  })
+
+  it('accepts array with falsy values', () => {
+    expect(listFormat([null, undefined, false, NaN, '', 0, -0])).toBe(
+      ', 0 og 0'
+    )
+  })
+
+  it('accepts array with string values', () => {
+    expect(listFormat(['Foo', 'Bar', 'Baz'])).toBe('Foo, Bar og Baz')
+  })
+
+  it('accepts array with empty string values', () => {
+    expect(listFormat(['', '', ''])).toBe(',  og ')
```

Comment:
```suggestion
    expect(listFormat(['', '', ''])).toBe('')
```
Should this result in an empty string or  '`,  og '` ? 🤔 
What do we think?

Reply:
I don't think so. This edge case needs to be handled by the application. Also because this is what we get from the browser API. 

An app can do this `list.filter(Boolean)` if there is the potential that a value will be falsy.